.rubocop_todo.yml

```diff
@@ -57,18 +57,11 @@ Metrics/ParameterLists:
 RSpec/DescribeClass:
   Enabled: false
 
-# Offense count: 87
+# Offense count: 224
```

Comment:
i'm happy to bump this to a higher and more reasonable default if people want, either in a commit in this PR or in a separate PR.

this update came because i re-generated this TODO file after adding the setting override i made in `.rubocop.yml` -- it doesn't have anything to do directly with my changes in this PR.

the only reason i haven't already added an override for this rule in this PR is that i know some of us find changes like that to be extra noise when reviewing.